upgrade/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if (NOT EMIL_BUILD_UPGRADE)
```

Comment:
As already stated above: I would exclude the Protobuf key target under pack only.